src/js/outreach/outreach.scss

```diff
@@ -100,14 +100,14 @@ html, body, .app-root {
 .dialog__button {
   font-size: 16px;
   margin-top: 40px;
-  padding: 10px 16px;
+  padding: 8px 48px;
   width: 100%;
 
   .fa-right-to-bracket {
     font-size: 20px;
     margin-left: 8px;
     vertical-align: text-top;
-    width: auto;
+    width: 20px;
```

Comment:
I don't love adding a fixed px width here, but we seem to do it where there's a `font-size: 20px` icon